color-eyre/Cargo.toml

```diff
@@ -18,7 +18,7 @@ issue-url = ["url"]
 track-caller = []
 
 [dependencies]
-eyre = "0.6.1"
+eyre = { path = "../eyre" }
```

Comment:
Please have a version too in here. That way, we get best of both worlds.

Reply:
Thank you @pksunkara — I added version to this dependency description as "1.0.0", is it okay?

Reply:
Yes, but please reset the lockfile to remove unneeded upgrades. And then run `cargo update` to update eyre related changes.

Reply:
That's done @pksunkara